fvtest/threadtest/abortTest.cpp

```diff
@@ -167,7 +167,7 @@ sleepingMain(void *arg)
 
 	EXPECT_EQ(rc, J9THREAD_PRIORITY_INTERRUPTED);
 
-	return rc;
+	return 0;
```

Comment:
Why not `static_cast<int>(rc)`? Or if we really don't care about the return value why not make the function `void`?